dashboard/test/ui/features/spritelab/eyes.feature

```diff
@@ -0,0 +1,12 @@
+@eyes
+@dashboard_db_access
+@as_student
+Feature: Sprite Lab Eyes
+
+Scenario: Basic Sprite Lab level
+  When I open my eyes to test "sprite lab eyes"
+  And I start a new Sprite Lab project
+  Then I see no difference for "initial load"
+  And I've initialized the workspace for the sample Sprite Lab project
+  Then I see no difference for "preview"
```

Comment:
Would it be useful to also 'run' the project? It could let us know if there are any issues that prevent students from clicking the run button and executing the code, but it might also introduce timing variance into this test. It might depend on what the starter code is.